### DISCUSSION TITLE
Unstable/Faction clothing stat change feedback

### DISCUSSION THREAD

**Discussion by PokkeTigrex** [FakeFishGames/Barotrauma#10289] [Unstable feedback]
![image](https://user-images.githubusercontent.com/85914375/199852306-22a913ed-c432-44c9-926c-f0b889c88236.png)

![image](https://user-images.githubusercontent.com/85914375/199852336-069eba75-ed07-4e95-b957-bfab40c5ee69.png)

With the recent unstable update, every clothing item in the game now contains some form of bonus. I think the changes are fairly good, but there could be some improvements. A good amount of colony clothes are either identical, or more inferior to starter clothes, outside one unique outfit being the medical officer uniform with its +15 medical bonus. Especially for clothes like researcher's outfit with almost similar stats to assistant without the +5 to weapon and helm. There can be more variety to stat distribution in the colony outfits so they aren't a complete downgrade, but a sidegrade or alternative.

**Comment by ubertpendragon** [comment]
`  - All starter clothing gives less protection now, while some shop/npc clothing now gives some benefit.`

**Comment by PokkeTigrex** [comment]
I'm not particularly sure why this became a discussion since a commit was pushed out which vastly altered the clothing balance as a whole relating to the original issue, but I might as well change it into a discussion on the changes and how agreeable the changes are and what can be changed for it, and swapped the category to unstable. I will update the main post when I have time with my thoughts on it. In the meantime, feel free to discuss here about the changes.
